.eslintrc.js

```diff
@@ -36,7 +36,8 @@ module.exports = {
       }
     ],
     "react/react-in-jsx-scope": "off",
-    "react/jsx-props-no-spreading": "off"
+    "react/jsx-props-no-spreading": "off",
+    "react/require-default-props": 0,
```

Comment:
Default props should not be required IMO, but open to a discussion if you disagree.

Reply:
I think it makes our code more self documenting. So its nice to have.